src/main/resources/db/migration/V18_01__add_veileder_historikk.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE VEILEDER_HISTORIKK (
+  id                        SERIAL             PRIMARY KEY,
+  uuid                      CHAR(36)           NOT NULL UNIQUE,
+  person_oversikt_status_id INTEGER REFERENCES PERSON_OVERSIKT_STATUS (id) ON DELETE CASCADE,
+  fra_dato                  DATE               NOT NULL,
+  tildelt_veileder          VARCHAR(7)         NOT NULL,
+  tildelt_enhet             VARCHAR(4)         NOT NULL,
+  tildelt_av                VARCHAR(7)
+);
+
+CREATE INDEX IX_VEILEDER_HISTORIKK_STATUS_ID ON VEILEDER_HISTORIKK (person_oversikt_status_id);
+
+INSERT INTO VEILEDER_HISTORIKK (uuid,person_oversikt_status_id,fra_dato,tildelt_veileder,tildelt_enhet)
```

Comment:
Initialiserer med eksisterende veilederknytninger. `fra_dato = oppfolgingstilfelle_start` blir litt omtrentlig.

Reply:
Ville `tildelt_dato` vært mer naturlig navn på kolonnen enn `fra_dato` ? 
Også litt usikker på om vi bør sette den datoen basert på noe (som oppfolgingstilfelle) eller om den bør være `null` for de ve ikke vet noe om :thinking: Bør kanskje ta en prat om det.

Reply:
Ja, vi vil jo ha data her i forsettelsen, så litt dumt at ting må være nullable bare fordi vi mangler data å initialisere med for eksisterende tilordninger. Og så er jo dette det man typisk vil sortere på når man skal vise historikken.